safekit-cluster-mirror/nestedtemplates/cfgMirror.json

```diff
@@ -0,0 +1,94 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"vmname": {
+			"type": "string",
+			"metadata": {
+				"description": "vm name"
+			}
+		},
+		"ostype": {
+			"type": "string",
+			"metadata": {
+				"description": "OS type (windows, linux)"
+			},
+			"defaultValue": "windows"
+		},
+		"moduleName": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description": "url of the application module to install on all nodes"
+			}
+		},
+		"location": {
+			"type": "string",
+			"metadata": {
+				"description": "resources location"
+			},
+			"defaultValue": "[resourceGroup().location]"
```

Comment:
remove defaultValue since this must be passed through

Reply:
OK